scrapper.py

```diff
@@ -26,10 +40,20 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        url = 'https://vz.ru/news/'
```

Comment:
good, but all this should be written under the find_articles function. extract_url extracts specific url only

Reply:
I am sorry, something happened with index.html when I pushed changes in scrapper.py, can you please tell me how can I fix it (when I open my pull request, I see "HTML" in languages)?